generator/discovery/update_discovery_doc.sh

```diff
@@ -35,8 +35,21 @@ readonly COMPUTE_DISCOVERY_JSON_RELATIVE_PATH="generator/discovery/compute_publi
 io::log_h2 "Fetching discovery document from ${COMPUTE_DISCOVERY_DOCUMENT_URL}"
 curl "${COMPUTE_DISCOVERY_DOCUMENT_URL}" >"${PROJECT_ROOT}/${COMPUTE_DISCOVERY_JSON_RELATIVE_PATH}"
 
+REVISION=$(sed -rn 's/  \"revision\": \"([[:digit:]]+)\",/\1/p' "${PROJECT_ROOT}/${COMPUTE_DISCOVERY_JSON_RELATIVE_PATH}")
```

Comment:
This looks like our first use of extended REs in sed, so I might follow the man page's advice and "for portability use POSIX -E" instead of -r.